hof.settings.json

```diff
@@ -1,11 +1,12 @@
 {
-  "theme": "govuk",
+  "appName": "UKVI Complaints",
+  "theme": "govUK",
   "routes": [
     "./apps/ukvi-complaints"
   ],
-  "views": "./apps/ukvi-complaints/views",
+  "views": ["./apps/ukvi-complaints/views"],
   "session": {
-    "name": "hod.sid",
+    "name": "ukvi-complaints.hof.sid",
```

Comment:
On the /cookies page the session cookie is listed as hod.sid - one of these may need to be changed